components/steps/style/progress-dot.less

```diff
@@ -17,7 +17,7 @@
       }
     }
     &:first-child .@{steps-prefix-cls}-icon-dot {
-      left: 2px;
+      left: 0;
```

Comment:
look normal, but we can remove this rule?

Reply:
> look normal, but we can remove this rule?

Yes, I'll submit it again